Core/Models/DynamicsBase.hpp

```diff
@@ -108,6 +108,11 @@ class DynamicsBase{
     [[nodiscard]] virtual inline bool DynamicsArePaused(void) const{
         return dynamics_paused_;
     }
+    // Calling GetPosition returns the quaternion as a Vector4
+    [[nodiscard]] virtual inline Quaternion GetOrientation(void) const
+    {
+        return Quaternion(position_.data());
+    }
```

Comment:
I am hesitant about adding this member function to the base class, since then it would imply that any dynamics object has an orientation. (E.g., if we're using this for a 5 dof joint space admittance control, what would the orientation even mean?) This member function only makes sense when `DynamicsBase` is an `RotationSecondOrder`, so adding it here feels too restrictive. 

I understand your reasoning for doing this though. I think in the VRex code, you could instead do something like:
```
Eigen::Quaternion(model.GetPosition().data());
```
and this would be equivalent and would not require modifications to the base class.